product_docs/docs/epas/15/installing/uninstalling/linux_uninstall.mdx

```diff
@@ -51,25 +51,26 @@ To uninstall EDB Postgres Advanced Server and its dependent packages, use the ap
 -   On RHEL or CentOS 7:
 
     ```text
-    yum remove edb-as15-server*
+    yum remove edb-as<xx>-server*
     ```
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
```

Comment:
left angle brackets outside of code formatting cause the build to fail. Options include enclosing <*xx*> in tick marks to format it as code, remove the angle brackets, or use the ASCII code for the left angle bracket (&lt;)
```suggestion
Where <*xx*> represents the EDB Postgres Advanced Server version number.
```

Reply:
Thanks for pointing that out! I checked out a couple of other products, and there isn't a consistent approach. Some use <x>, some <xx>, and some <X>. I think since we have a two-digit version number we should use `<xx>`. I checked with Betsy, and she told me to check with you guys :-)

Reply:
I agree. Let's use `<xx>` like we do in the EPAS install topics:
![image](https://github.com/EnterpriseDB/docs/assets/83650384/05b8d2c4-5727-49a5-8621-0db82fb774ef)

Reply:
Will do! Thanks!